### PR TITLE
added base64 option

### PR DESCRIPTION
**Pull request by MJam9**
base64 coded option implementit

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -31,7 +31,8 @@ ENV DEV_APP_SKEY 00000000000000000000000000000010
 ENV DEV_APP_KEY 00000000000000000000000000000100
 ENV DEV_NAME VirtualSensor
 ENV DEV_MYTYPE ConfirmedDataUp
-ENV DEV_PAYLOAD hello
+ENV DEV_PAYLOAD E4gBAHsAAGRJEGU=
+ENV DEV_BASE true
```
Comment (ttulka) [inline Dockerfile]: DEV_BASE -> DEV_PAYLOAD_BASE64

File: Dockerfile [hunk 2]
```diff
@@ -31,7 +31,8 @@ ENV DEV_APP_SKEY 00000000000000000000000000000010
 ENV DEV_APP_KEY 00000000000000000000000000000100
 ENV DEV_NAME VirtualSensor
 ENV DEV_MYTYPE ConfirmedDataUp
-ENV DEV_PAYLOAD hello
+ENV DEV_PAYLOAD E4gBAHsAAGRJEGU=
```
Comment (ttulka) [inline Dockerfile]: hello

File: Dockerfile (hunk 1 above)
Comment (ttulka) [reply Dockerfile]: default false

**Review by ttulka — CHANGES_REQUESTED** [review]

**Comment by MJam9** [comment]
set hello

**Pull request merged**